.packit.yaml

```diff
@@ -16,8 +16,7 @@ jobs:
   - job: copr_build
     trigger: pull_request
     enable_net: true
-    # keep in sync with https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next
-    targets: &copr_targets
+    targets:
       - fedora-rawhide-aarch64
```

Comment:
no ppc64le/s390 here?

Reply:
so ppc64le/s390x builds take really long to complete. Adding them on `trigger: pull_request` would PRs could take forever to merge. That's why I've only added them to `trigger: commit`.